.github/workflows/smoke-test.yaml

```diff
@@ -60,49 +62,43 @@ jobs:
       - name: Run ${{ matrix.target.name }}
         run: |
           kubectl apply -f ${{ matrix.target.template }}
-          kubectl wait -n kube-system --for=condition=Ready --all pod --timeout=5m
+          kubectl wait -n kube-system --for=condition=Ready --all pod --timeout=${{ env.TIMEOUT }}
```

Comment:
Can you split this into a separate commit and explain why the default timeout is changed from 5m to 2m?

Reply:
makes sense :+1: